### ISSUE TITLE
Support for influxdb v2?

### ISSUE THREAD

**Issue by doctarrwho** [tedpearson/electric-usage-downloader#10]
Any chance you can add support for influxdb v2? It requires specifying the Org and Bucket ID/name.

```
Authenticating with SmartHub API...
Fetching data from SmartHub API...
Data not ready, retrying...
Data received, transforming...
Writing data to database...
panic: invalid: Please provide either orgID or org

goroutine 1 [running]:
main.main()
        /repos/electric-usage-downloader/main.go:9 +0x33
exit status 2
```

**Comment by tedpearson** [comment]
I added support for org, which is specified in the config file. Note that "database" in the config file is actually "bucket" for influxdb v2. Give it a try!

**Comment by tedpearson** [comment]
(Available in v2.2.0 release https://github.com/tedpearson/electric-usage-downloader/releases/tag/v2.2.0)
